cmd/llamacc/main.go

```diff
@@ -293,7 +293,7 @@ func main() {
 		if ex, ok := err.(*exec.ExitError); ok {
 			os.Exit(ex.ExitCode())
 		}
-		fmt.Fprintf(os.Stderr, "Running gcc: %s\n", err.Error())
+		fmt.Fprintf(os.Stderr, "Running llamacc locally: %s\n", err.Error())
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "Running %s locally: %s\n", cc, err.Error())
```

In this case we're not going to the cloud or really doing anything Llama-specific, so let's provide a bit clearer explanation.